.env

```diff
@@ -23,9 +23,13 @@ HTTP_GW_VERSION=0.16.0
 HTTP_GW_IMAGE=nspccdev/neofs-http-gw
 
 # S3 Gate
-S3_GW_VERSION=0.17.0
+S3_GW_VERSION=0.17.0-15-gda9f272
```

Comment:
Debug?

Reply:
I've thought we need to update s3-gw  image (by new release or intermediate image) to be able to check coredns works together with s3-gw correctly. Though we can check coredns separately (making dns request to `coredns` container).

Reply:
I don't mind to bump version of S3 gateway, we use intermediate version in dev-env so it is okay, but it should be available on DockerHub.